drivers/fpga/imx_m4.c

```diff
@@ -397,6 +391,13 @@ int fpga_loadbitstream(int d, char *bitstream, size_t size, bitstream_type t)
 		return ret;
 	}
 #endif
+
+	/* check if the M4 is already running */
```

Comment:
looks good to me - I guess this value is always 0 in single boot mode? also across warm resets?

Reply:
It's 0 actually on a warm reset.. which means we would perform a hot update of the QSPI if the SHA was different ... but now that we're known running in TCM.. I'm not sure it matters?